internal_packages/composer/lib/send-action-button.jsx

```diff
@@ -116,19 +116,24 @@ export default class SendActionButton extends React.Component {
     return {preferred, rest};
   }
 
-  _contentForAction = ({iconUrl}) => {
-    let plusHTML = "";
-    let additionalImg = false;
+  _contentForAction = ({buttonTitle, title, iconUrl}) => {
+    let additionalImg = null;
+
+    const actionTitle = buttonTitle || title;
 
     if (iconUrl) {
-      plusHTML = (<span>&nbsp;+&nbsp;</span>);
-      additionalImg = (<RetinaImg url={iconUrl} mode={RetinaImg.Mode.ContentIsMask} />);
+      additionalImg = (
+        <span className="additional">
+          &nbsp;
+          <RetinaImg url={iconUrl} mode={RetinaImg.Mode.ContentIsMask} />
+        </span>
+      );
     }
 
     return (
       <span>
         <RetinaImg name="icon-composer-send.png" mode={RetinaImg.Mode.ContentIsMask} />
-        <span className="text">Send{plusHTML}</span>{additionalImg}
```

Comment:
You removed "Send", but aren't setting it the default anywhere. If you don't have the Send+Archive option/extension enabled, will the button just be blank?